packages/estatico-boilerplate/gulpfile.js

```diff
@@ -432,7 +434,7 @@ gulp.task('js:test', (done) => { // eslint-disable-line consistent-return
   });
 
   tests.stderr.on('data', (data) => {
-    if (`${data}`.match(/Test Suites: (.*?) failed/m)) {
+    if (stripAnsi(`${data}`).match(/(Test Suites: (.*?) failed|npm ERR!)/m)) {
```

Comment:
This will get rid of all the color instructions in the log, making it easier to match `npm ERR`, e.g.